packages/gatsby-source-wordpress/src/steps/declare-plugin-options-schema.ts

```diff
@@ -538,9 +538,7 @@ When using this option, be sure to gitignore the wordpress-cache directory in th
       .items(
         Joi.object({
           search: Joi.string()
-            .description(
-              `The regex rule used to search a terme. Using a 'regular expression literal' is recommended over a simple regex string.`
-            )
+            .description(`The regex rule used to search a terme.`)
```

Comment:
```suggestion
            .description(`The regex rule used to search when replacing strings in node data. It will search stringified JSON of node to capture strings at any nested depth.`)
```